staging/src/k8s.io/apiserver/pkg/server/options/api_enablement.go

```diff
@@ -71,6 +74,18 @@ func (s *APIEnablementOptions) Validate(registries ...GroupRegisty) []error {
 		return append(errors, err)
 	}
 
+	if feature.DefaultFeatureGate.Enabled(features.RequestManagement) {
+		flowcontrolRegistered := false
+		for _, g := range groups {
+			if g == flowcontrolv1alpha1.SchemeGroupVersion.Group {
+				flowcontrolRegistered = true
+			}
+		}
+		if !flowcontrolRegistered {
```

Comment:
this only works right on the kube-apiserver.  move this check there.

Reply:
yes, KCM/scheduler shouldn't hit this code.